language_detector/main.py

```diff
@@ -2,10 +2,39 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    words = text.split()
+    count_dict ={}
+    
```

Comment:
I believe that this function (word_count) could live outside of the function detect_languages allowing use to apply to other functions if need be.